app/controllers/map.ts

```diff
@@ -13,8 +13,8 @@ export default class MapComponent extends Controller {
   @service declare router: RouterService;
   declare model: ModelFrom<MapRoute>;
 
-  @tracked locationData: any = this.model.locationData.toArray();
-  @tracked agendaData: any = this.model.agendaData.toArray();
```

Comment:
This can never have worked properly. Controllers are only instanced once and there is no `this.model` at construction time. It's only available after the route executes the `setupController` method. This change is probably not correct either, but digging into that code is out of scope for this PR 😬.